test/smoke_test/bcda_client.go

```diff
@@ -134,7 +134,12 @@ func writeFile(resp *http.Response, filename string) {
 	if err != nil {
 		panic(err)
 	}
-	defer out.Close() // #nosec G307
+	defer func() {
+		if ferr := out.Close(); ferr != nil {
+			panic(ferr)
```

Comment:
Maybe just log instead of panic here.  I think we'll get that for free, though, if we use a common utility function.